aws/cft.yml

```diff
@@ -0,0 +1,141 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Conjur CE Stack, backed by a PostgreSQL database on RDS
+
+Parameters:
+  ConjurAccount:
+    Type: String
+    Description: Name of Conjur account
+
+  ConjurAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: Password for Conjur admin account
+
+  DBAdminUser:
+    Type: String
+    Description: RDS instance admin user
+    Default: conjuradmin
+
+  DBAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: RDS instance admin password
+
+  ImageId:
+    Type: AWS::EC2::Image::Id
+    Description: Conjur CE AMI
+
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: AWS keypair to use
+
+  VpcId:
+    Type: AWS::EC2::VPC::Id 
+    Description: VPC in which to launch Conjur instances
+
+  VpcSubnetIds:
+    Type: "List<AWS::EC2::Subnet::Id>"
+    Description: Subnets in which to launch Conjur and Postgres. Specify two, in different Availability Zones.
+
+
+Metadata:
+  AWS::CloudFormation::Interface: 
+    ParameterGroups: 
+      - Label: 
+          default: "Conjur Configuration"
+        Parameters:
+          - ConjurAccount
+          - ConjurAdminPassword
+          - DBAdminUser
+          - DBAdminPassword
+      - Label: 
+          default: "EC2 Instance Configuration"
+        Parameters:
+          - ImageId
+          - KeyName
+          - VpcId
+          - VpcSubnetIds
+
+Resources:
+  ConjurInstance:
+    Type: "AWS::EC2::Instance"
+    Properties: 
+      ImageId: !Ref ImageId
+      InstanceType: t2.medium
+      KeyName: !Ref KeyName
+      SecurityGroupIds: [!Ref ConjurSecurityGroup]
+      SubnetId: !Select [0, !Ref VpcSubnetIds]
+      UserData:
+        Fn::Base64: !Sub |
+          #!/bin/bash -e
+
+          exec >>/var/log/conjur-init.log 2>&1
+
+          set -x
+          # Only initialize once
+          if grep -q CONJUR_DATA_KEY /etc/default/conjur; then 
+            exit 0;
+          fi
+
+          cat - >>/etc/default/conjur <<EOF
+          CONJUR_ACCOUNT=${ConjurAccount}
+          CONJUR_DATA_KEY=$(docker run --rm conjur data-key generate)
+          DATABASE_URL=postgres://${DBAdminUser}:${DBAdminPassword}@${RDSInstance.Endpoint.Address}/postgres
+          RAILS_ENV=production
+          EOF
+
+          admin_api_key="$(/opt/conjur/bin/server create | awk '/API key for admin/ {print $NF}')"
+          /opt/conjur/bin/server wait_for_server
+
+          /opt/conjur/bin/set_admin_password "$admin_api_key" "${ConjurAdminPassword}"
+
+
+  RDSInstance:
+    Type: "AWS::RDS::DBInstance"
+    Properties:
+      Engine: postgres
+      EngineVersion: 9.3.17
+      AllocatedStorage: 5
+      DBInstanceClass: db.t2.medium
+      MasterUsername: !Ref DBAdminUser
+      MasterUserPassword: !Ref DBAdminPassword
+      DBSubnetGroupName: !Ref DBSubnetGroup
+      VPCSecurityGroups:
+        - !Ref PostgresSecurityGroup
+
+  ConjurSecurityGroup:
```

Comment:
For our CFN templates, I think it's better to put the security groups up top, because that is the part most consumers will want to tweak. Different `CidrIp` for example. `Resources` should be in SGs, EC2, DB order i think.

Reply:
OK, will do.